CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
We don't have that cmake on CI. I'm not merging anything that breaks the CI, period.